### PR TITLE
Cpp14 support

### PR DESCRIPTION
**Pull request by marektamaskovic**

### CODE REVIEW

File: src/algorithm_advanced.cpp [hunk 1]
```diff
@@ -482,7 +489,12 @@ namespace st2se {
 
         std::string s;
 
-        if (auto _s = std::get_if<std::string>(&(in.value))) {
+        #if __cplusplus < 201703L
```
Comment (davidbolvansky) [inline src/algorithm_advanced.cpp:492]: you can use your variant_ns here :D

Comment (marektamaskovic) [reply src/algorithm_advanced.cpp:492]: Yeah I should fix it

**Comment by davidbolvansky** [comment]
Is this project still alive? :P

**Comment by marektamaskovic** [comment]
Yep, but there are other priorities right now.

**Pull request merged**